doc/website/getting-started/installation.md

```diff
@@ -67,9 +67,6 @@ We provide generic QNX SDP 7.0 toolchain files for ARM_64 and X86_64 in `./tools
 ARM_64: `./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qnx_sdp70_aarch64le.cmake`<br>
 X86_64: `./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qnx_sdp70_x86_64.cmake`
 
-!!! attention
-    Please make sure the directory `/var/lock` is created by your QNX image
```

Comment:
please make this a `Please ensure the filesystem on "/tmp" supports file locking`. This was the main issue why we switched to TCP port based locking

Reply:
Thanks for the hint! I think we should make the file locking directory configurable.

@elBoberido @mossmaurice what do you think about setting this in the config file and via a command line parameter?! The default should be `/tmp` but it also makes definitely sense to set it to `/var/lock`

Reply:
As written in [yesterdays comment](https://github.com/eclipse-iceoryx/iceoryx/pull/626/files#r599034627) QNX advises to use `/var/locks`. However, in the QNX image which @dkroenke used there was no such directory. I've pinged @anup-pem to clarify the directory for QNX:

Reply:
@mossmaurice do you mean `/var/lock` or `/var/locks`?

Reply:
I meant `/var/lock` ;)

Reply:
Had chat with @anup-pem yesterday. From his point of view `/var/lock` should be fine. IMHO `/tmp` should also work, but this needs to be tested. I've added a todo

Reply:
@mossmaurice Agree, `/tmp` should be fine unless you want it to be persistent across reboots. That said, should test it as you already mentioned.